modules/virt-enlarging-vm-disk-pvc.adoc

```diff
@@ -1,14 +1,14 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-vm-disk-resizing.adoc
+// * virt/virtual_machines/virtual_disks/virt-expanding-vm-disks.adoc
 
 :_content-type: PROCEDURE
-[id="virt-enlarging-vm-disk_{context}"]
-= Enlarging a virtual machine disk
+[id="virt-enlarging-vm-disk-pvc_{context}"]
+= Expanding a PVC
 
-Virtual machine (VM) disk enlargement makes extra space available to the virtual machine. However, it is the responsibility of the VM owner to decide how to consume the storage.
+You can increase the size of a virtual machine (VM) disk by expanding the persistent volume claim (PVC) of the disk.
 
-If the disk is a `Filesystem` PVC, the matching file expands to the remaining size while reserving some space for file system overhead.
+If the PVC uses the file system volume mode, the disk file expands to the available size while reserving some space for file system overhead.
```

Comment:
The difference between filesystem volume mode and block volume mode, is that with a filesystem PVC we have a filesystem with disk image file on it, where with a block volume mode, we use the block device directly.

`the disk IMAGE file expands...`